noir/noir-repo/acvm-repo/acvm/tests/solver.rs

```diff
@@ -105,8 +105,13 @@ fn inversion_brillig_oracle_equivalence() {
     ])
     .into();
     let unconstrained_functions = vec![];
-    let mut acvm =
-        ACVM::new(&StubbedBlackBoxSolver, &opcodes, witness_assignments, &unconstrained_functions);
+    let mut acvm = ACVM::new(
+        &StubbedBlackBoxSolver,
+        &opcodes,
+        witness_assignments,
+        &unconstrained_functions,
+        Default::default(),
```

Comment:
I like to avoid usage of `Default::default()` whereever possible as it removes any type information. This would be much clearer as `&[]`